script-test/playbackstrategies/msestrategytest.js

```diff
@@ -535,5 +547,61 @@ require(
           });
         });
       });
+
+      describe('getMediaPlayerInfo', function () {
+        var mockEvent = {
+          mediaType: 'video',
+          oldQuality: 0,
+          newQuality: 1,
+          type: 'qualityChangeRendered'
+        };
+
+        mockPluginsInterface = jasmine.createSpyObj('interface', ['onErrorCleared', 'onBuffering', 'onBufferingCleared', 'onError', 'onFatalError', 'onErrorHandled', 'onPlayerInfoUpdated']);
+
+        mockPlugins = {
+          interface: mockPluginsInterface
+        };
+
+        injector.mock({'bigscreenplayer/plugins': mockPlugins});
+
+        it('should call plugins with playback bitrate', function () {
+          setUpMSE();
+          mockDashInstance.getBitrateInfoListFor.and.returnValue([{bitrate: 1000}, {bitrate: 2000}, {bitrate: 3000}]);
+          mseStrategy.load(null, null, 0);
+
+          dashEventCallback(dashjsMediaPlayerEvents.QUALITY_CHANGE_RENDERED, mockEvent);
+
+          expect(mockPluginsInterface.onPlayerInfoUpdated).toHaveBeenCalledWith({
+            downloadBitrate: undefined,
+            playbackBitrate: 2,
```

Comment:
Small point, but I had to go on a bit of a journey to see where this number `2` came from. Perhaps a comment could be added, or have the second bitrate something like 2048 and then expect 2.048, so it's easier to make the mental jump.

Reply:
Changed it to your suggestion, hopefully it makes it more easily understood